edx_exams/apps/access/views.py

```diff
@@ -0,0 +1,63 @@
+"""
+Exam Access Ticket Views
+"""
+from datetime import datetime, timedelta
+
+from django.http import HttpResponse
+from rest_framework import status
+from rest_framework.views import APIView
+from token_utils.sign import create_jwt
+
+from edx_exams.apps.core.models import ExamAttempt
+
+
+class ExamAccessTicketsView(APIView):
+    """
+    todo: add docs after impl is complete
+    """
+
+    # todo: should this be a get request?
+    def get(self, request):
+        user = request.user
+        exam = request.exam
+        claims = {}
+        claims.update(exam.course_id)
+        claims.update(exam.content_id)
+        exp_secs = 60
+
+        exam_attempt = ExamAttempt.get_latest_user_exam_attempt(user, exam)
+        # todo: should this be called ticket or token? For consistency?
+        cookie_name = "exam_access_ticket"
```

Comment:
Question: should we call them exam_access_tickets? Or tokens? Just want to keep it consistent.

Reply:
I think we landed on using 'ticket' for this to avoid ambiguity with existing JWT 'tokens' so I'd go with that.

Reply:
I think it was actually the other way around, I used "ticket" all over my confluence doc but people didn't like it hence token in the library name.

It doesn't actually matter but yes it should be consistent 😆